packages/airnode-validator/src/commands/utils.ts

```diff
@@ -45,6 +47,13 @@ conversionTemplates.forEach((file) => {
   conversions[fromName][fromVersion][toName].push(toVersion);
 });
 
+// Regular expression that does not match anything, ensuring no escaping or interpolation happens
+// https://github.com/lodash/lodash/blob/4.17.15/lodash.js#L199
+const NO_MATCH_REGEXP = /($^)/;
+// Regular expression matching ES template literal delimiter (${}) with escaping
+// https://github.com/lodash/lodash/blob/4.17.15/lodash.js#L175
+const ES_MATCH_REGEXP = /\$\{([^\\}]*(?:\\.[^\\}]*)*)\}/g;
```

Comment:
As I mentioned before, I don't like that you just copy&pasted the regular expressions and code from the `airnode-node` package. Feel free to refactor the code there so you can import just the interpolation itself or move the implementation here and import it in the `airnode-node` package but the code should be only in one place.

Reply:
Removed in 538ee33